resotocore/resotocore/__main__.py

```diff
@@ -223,6 +226,7 @@ async def on_start() -> None:
         await log_ship.start()
         await api.start()
         await infra_apps_package_manager.start()
+        await graph_manager.start()
```

Comment:
Please also add stop method and call it here. Even if there is currently no stop action.